smartmin/views.py

```diff
@@ -368,19 +368,21 @@ def as_json(self, context):
         Responsible for turning our context into an dict that can then be serialized into an
         JSON response.
         """
-        return context
+        raise NotImplementedError("this view can't be rendered as JSON")
 
     def render_to_response(self, context, **response_kwargs):
         """
         Overloaded to deal with _format arguments.
         """
-        # should we actually render in json?
+        # should we try rendering as JSON?
         if '_format' in self.request.GET and self.request.GET['_format'] == 'json':
```

Comment:
This really should be changed to just look at the accept header

Reply:
True tho I think we still have one place in RP where we use `_format=select2`